var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -539,6 +538,11 @@ def define_enable(suffix, value=None):
             define_from_variant('Amesos2_ENABLE_LAPACK', 'amesos2'),
         ])
 
+        if '+aztec' in spec and '+triutils' in spec:
```

Comment:
The correct way to handle this is to add a conflict saying "you cannot have aztec enabled and triutils disabled":
```fortran
conflicts('+aztec', when='~triutils')
```

Otherwise users who want aztec will find it silently missing!